### DISCUSSION TITLE
Download warning for images that are marked Unusable, and/or have No Rights

### DISCUSSION THREAD

**Discussion by abdelrahmansd** [guardian/grid#3533] [Ideas]
When Downloading an image that is marked Unusable, and/or has No Rights, BBC would like to add additional usage warnings 

Given a <user> selects 'download' for an image 

And the Rights and Restrictions are set to "None"  ( image has red warning triangle)

When the user clicks the download button

Then the user should see an alert that tells them that the image has 'No Rights'

And the user must interact with the alert to confirm that they understand that the image has no usage rights

**_` You are about to download an image that has NO agreed right for usage anywhere`_**

and This action will be recorded

and that action will be  for everyone 

and The downloads are recorded in the Usages tab of the image,

----------------------------------------------------
the [React2Angular](https://www.npmjs.com/package/react2angular) npm library will be used to achieve this feature

**Comment by paperboyo** [comment]
I understand this is needed until there is proper integration with tools on your side (which prevents cropping these, hence using this where there is proper integration). We also have a small minority of users who may be downloading images. Most of them will use crops URL, so this won’ help, but also doesn’t look like something that will cause any problems, so we are happy for this to be a general feature (not behind config etc.).

What should happen if the batch of downloaded images contain **some** images like this? Do we need a warning then to say that only some are like this?

**Reply by honorcb** [reply]
That would be reasonable,  I had forgotten you can download a selection when writing the ticket.

**Comment by paperboyo** [comment]
Much harsher version (blocking non-free downloads for unpermissioned users) implemented behind permission in https://github.com/guardian/grid/pull/3873.
